### PR TITLE
install nebula graph by rpm or deb

### PR DESCRIPTION
**Pull request by izhuxiaoqing**

### CODE REVIEW

File: docs-2.0/4.deployment-and-installation/2.compile-and-install-nebula-graph/2.install-nebula-graph-by-rpm-or-deb.md [hunk 1]
```diff
@@ -2,130 +2,133 @@
...
-    URL：
+    URL:
 
     ```bash
-    //Centos 6 
-    https://oss-cdn.nebula-graph.io/package/<release_version>/nebula-graph-<release_version>.el6.x86_64.rpm
-
     //Centos 7
-    https://oss-cdn.nebula-graph.io/package/<release_version>/nebula-graph-<release_version>.el7.x86_64.rpm
+    https://oss-cdn.nebula-graph.com.cn/package/<release_version>/nebula-graph-<release_version>.el7.x86_64.rpm
```
Comment (whitewum) [inline docs-2.0/4.deployment-and-installation/2.compile-and-install-nebula-graph/2.install-nebula-graph-by-rpm-or-deb.md:23]: for english doc. the site is .io
for chinese doc the site is .com.cn

File: docs-2.0/4.deployment-and-installation/2.compile-and-install-nebula-graph/2.install-nebula-graph-by-rpm-or-deb.md [hunk 2]
```diff
@@ -75,57 +71,64 @@ Prepare the right [resources](../1.resource-preparations.md).
     https://oss-cdn.nebula-graph.io/package/v2-nightly/<yyyy.mm.dd>/nebula-graph-<yyyy.mm.dd>-nightly.ubuntu1804.amd64.deb
 
     //Ubuntu 2004
-    https://oss-cdn.nebula-graph.io/package/v2-nightly/<yyyy.mm.dd>/nebula-graph-<yyyy.mm.dd>-nightly.ubuntu2004.amd64.deb
+    https://oss-cdn.nebula-graph.io/package/v2-nightly/<yyyy.mm.dd>/nebula-graph-<yyyy.mm.dd>-nightly.ubuntu1804.amd64.deb
```
Comment (whitewum) [inline docs-2.0/4.deployment-and-installation/2.compile-and-install-nebula-graph/2.install-nebula-graph-by-rpm-or-deb.md:74]: ubuntu2004

Comment (whitewum) [reply docs-2.0/4.deployment-and-installation/2.compile-and-install-nebula-graph/2.install-nebula-graph-by-rpm-or-deb.md]: this is chinese version error. i have fixed it.

**Review by randomJoe211 — APPROVED** [review]

File: docs-2.0/4.deployment-and-installation/2.compile-and-install-nebula-graph/2.install-nebula-graph-by-rpm-or-deb.md [hunk 3]
```diff
@@ -78,54 +74,61 @@ Prepare the right [resources](../1.resource-preparations.md).
     https://oss-cdn.nebula-graph.io/package/v2-nightly/<yyyy.mm.dd>/nebula-graph-<yyyy.mm.dd>-nightly.ubuntu2004.amd64.deb
     ```
 
-    For example, download the `Centos 7.5` package developed and built in `2021.03.28`：
+    For example, download release package `2.x` for `Centos 7.5` in `2021.03.28`:
```
Comment (foesa-yang) [inline docs-2.0/4.deployment-and-installation/2.compile-and-install-nebula-graph/2.install-nebula-graph-by-rpm-or-deb.md:77]: `2021.08.18`

**Review by whitewum — APPROVED** [review]

**Review by foesa-yang — APPROVED** [review]

**Pull request merged**